.github/workflows/build-test-e2e.yml

```diff
@@ -5,7 +5,7 @@ env:
   BUILD_ARTIFACT_NAME: swc-war
   BUILD_DIR: target
   BUILD_NAME: portal-develop-SNAPSHOT.war
-  NODE_VERSION: 18.16.0
+  NODE_VERSION: 22
```

Comment:
Update Node to 22 (latest LTS) since it includes the latest version of [corepack](https://nodejs.org/api/corepack.html), which we will use to install pnpm.